lib/hq/graphql/input_object.rb

```diff
@@ -32,19 +32,19 @@ def format_nested_attributes
       end
 
       #### Class Methods ####
-      def self.with_model(model_name, attributes: true, associations: false, enums: true)
+      def self.with_model(model_name, excluded_inputs = [], attributes: true, associations: false, enums: true)
         self.model_name = model_name
         self.auto_load_attributes = attributes
         self.auto_load_associations = associations
         self.auto_load_enums = enums
 
         lazy_load do
           model_columns.each do |column|
```

Comment:
I would wrap in the global inputs here to avoid any load order issues

```suggestion
        lazy_load do
          excluded_inputs += ::HQ::GraphQL.excluded_inputs
          model_columns.each do |column|
```